docs/api-reference.md

```diff
@@ -51,7 +51,15 @@ Similar to connection-builder but is used to build the JDBC properties file. For
 {"server" : "myserver.somewhere.net", "username" : "myusername", "password" : "mypassword"}
 ```
 
-**Return:** vector of formatted key=value pairs that will be written to the properties file
+**Return:** dictionary or vector
+
+Recommended: a dictionary of key/value pairs that will be written to the properties file
+
+```javascript
+["UID" : "myusername", "Host" : "myserver.somewhere.net", "PWD" : "mypassword"];
+```
+
+Since 2019.4.1: vector of formatted key=value pairs that will be written to the properties file
```

Comment:
Isn't the dictionary the new return type for 2019.4.1+ ?

Reply:
Right. I think I made a mistake here. How about I change the description as below? Do you think that makes sense?

a dictionary of key/value.... (recommended and available since 2019.4.1)
...
vector of formatted key=value pairs ...
...

Reply:
Looks great.